main.c

```diff
@@ -0,0 +1,212 @@
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Why is it needed?

Reply:
#include <stdio.h> header file is required for standard input/output operations. it provides the declarations for functions like printf(), scanf()
#include <stdlib.h> This header file is required for memory allocation and deallocation ,this header file is necessary in programs that work with data structures like Array, linkedlist etc